src/lib/Timeline.js

```diff
@@ -208,6 +209,7 @@ export default class ReactCalendarTimeline extends Component {
     itemTouchSendsClick: false,
 
     style: {},
+    className: 'react-calendar-timeline',
```

Comment:
`react-calendar-timeline` should always exist on className I would suggest appending whatever class passed to `react-calendar-timeline`

Reply:
Updated; left the prop name as `className` which matches the other components' props, and updated the docs.